lib/command/src/Obelisk/Command.hs

```diff
@@ -181,6 +183,8 @@ data DeployInitOpts = DeployInitOpts
   { _deployInitOpts_outputDir :: FilePath
   , _deployInitOpts_sshKey :: FilePath
   , _deployInitOpts_hostname :: [String]
+  , _deployInitOpts_sslHostname :: String
+  , _deployInitOpts_adminEmail :: String
```

Comment:
What is with all the use of `String` here?  Why not `Text`?

Reply:
This whole code base could use a revamp to use `Text`. `process` uses `String` for all its arguments and I think that bled all the way up to our types.

Reply:
Yup, I used `String` only to be consistent. I can open a separate PR that converts the whole code base to use `Text`.

Reply:
@3noch I understand how that can happen, but it's a symptom of using the wrong approach.  People should always keep the core of their code in the best types available, and always do conversions to worse types at the edge of the system.  In other words, convert to "better" types at the earliest possible time, and convert to "worse" types at the last possible moment.  So, `String` should never pollute datastructures like this that we are writing.

@srid That sounds excellent; thanks!

Reply:
@srid And I 100% agree about keeping consistent with the surrounding code.

Reply:
@ryantrinkle I very much agree. I was tempted to make this transition multiple times but realized that using both `Text` and `String` in unpredictable places would be worse than just being consistent and doing a big switch at some point.